### ISSUE TITLE
PuTTY Session Profile reading from... somewhere I don't know

### ISSUE THREAD

**Issue by Zxurian** [jimradford/superputty#585]
So I'm using a portable version of SuperPutty (unzipped to a google drive folder, settings pointed to a subfolder within that). I'm also using KiTTY in an alternate folder wtihin google drive, also  in portable mode.

When within SuperPuTTY, (and I cleared all settings from super putty that I could find, including file  placed at %CURRENT_USER%\SuperPuTTY.settings), if I create a new session, The options under "PuTTY Session Profile" don't match what's in KiTTY. I did have putty on this system at one point, but I've gone through and removed the registry keys under Simon so I'm not sure where they're coming from.

Bottom line is for me to use superputty, I need it to read the sessions from kitty, but it doesn't appear to be doing that

**Comment by hnz101** [comment]
You are using KiTTY in portable mode? SuperPuTTy doesn't read the portable saved Sessions from the kitty.ini.
I would also very much like support for that case. So you can use SuperPuTTy together with kitty in a real portable setup.
Issue  #555 is about the same Problem I guess.

**Comment by Zxurian** [comment]
so if that were the case, then SuperPuTTY shouldn't be readying any sessions, but it is, and I can't figure out where they're reading from.

**Comment by waukeerules** [comment]
It's reading the local KiTTY file on the particular machine you're using. This would be your "Default Setting" that's created when KiTTY is first opened. My advice for you is to download KiTTY classic, and open the exe file within a separate directory than the portable version of KiTTY. Then re-create all your session names used in KiTTY portable. The session name is all that matters, your settings don't need to match in anyway. I use this exact set up at work.

Let me know if you have any questions.

**Comment by theyeag** [comment]
Here is a quick workaround that I found convenient. Open a command prompt and from the directory that holds kitty.exe run:
kitty.exe -convert-reg

I'm not sure what version of kitty this was implemented in but it works well for me. I would be careful about using this on unfamiliar machines unless you know what is being stored.
Found in a forum here: http://www.9bis.net/kitty/?action=forum&id=0054

I too would like to see superputty support sessions from portable instances of PuTTY/KiTTY.
